---
description: LOCALDB_ERROR_CANNOT_ACCESS_INSTANCE_FOLDER
title: LOCALDB_ERROR_CANNOT_ACCESS_INSTANCE_FOLDER | Документация Майкрософт
ms.custom: ''
ms.date: 03/04/2017
ms.prod: sql
ms.reviewer: ''
ms.technology: performance
ms.topic: reference
ms.assetid: c4288265-43f7-44fd-acab-ba64d04264d9
author: stevestein
ms.author: sstein
ms.openlocfilehash: 59321d66a5dd290fe6f1c148a60c9a1d0b2ecd68
ms.sourcegitcommit: e700497f962e4c2274df16d9e651059b42ff1a10
ms.translationtype: MT
ms.contentlocale: ru-RU
ms.lasthandoff: 08/17/2020
ms.locfileid: "88383290"
---
# <a name="localdb_error_cannot_access_instance_folder"></a>LOCALDB_ERROR_CANNOT_ACCESS_INSTANCE_FOLDER
 [!INCLUDE [SQL Server](../../includes/applies-to-version/sqlserver.md)]
    
## <a name="details"></a>Сведения  
  
|Категория|Значение|  
|-|-|  
|Название продукта|SQL Server|  
|Идентификатор события|261|  
|Источник события|Среда выполнения локальной базы данных SQL Server 12.0|  
|Компонент|API среды выполнения локальной базы данных|  
|Текст сообщения|Не удается получить доступ к папке экземпляра локальной базы данных:%% LOCALAPPDATA%% \ Microsoft\Microsoft SQL Server локальное \\ имя экземпляра дб\инстанцес<\> .|  
  
## <a name="explanation"></a>Объяснение  
 Не удалось получить доступ к папке экземпляра.  
  
## <a name="user-action"></a>Действие пользователя  
  
